docs/dev_guide/notebooks/transform.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<p>Remove reference to <code>unique</code> option since this will be removed soon </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>